profiles/arch/amd64/no-multilib/package.mask

```diff
@@ -23,6 +23,7 @@ mail-client/novell-groupwise-client
 
 # AMD64 Team <amd64@gentoo.org>
 # Mask packages that rely on amd64 multilib
+<=app-shells/scsh-0.6.9
```

Comment:
Can you tell me why `<app-shells/scsh-0.7.0` didn't work? What was wrong with it?

Reply:
> Can you tell me why `<app-shells/scsh-0.7.0` didn't work? What was wrong with it?

becasue _pre was lesser than full version

Reply:
> > Can you tell me why `<app-shells/scsh-0.7.0` didn't work? What was wrong with it?
> 
> becasue _pre was lesser than full version

That was my initial reaction too. Then I copied the same exact ebuild as `app-shells/scsh-0.7_p20160820` and got the following error:

```shell
!!! All ebuilds that could satisfy "scsh" have been masked.
!!! One of the following masked packages is required to complete your request:
- app-shells/scsh-0.7_p20160820::gentoo (masked by: package.mask)
/var/db/repos/gentoo/profiles/arch/amd64/no-multilib/package.mask:
# AMD64 Team <amd64@gentoo.org>
# Mask packages that rely on amd64 multilib

- app-shells/scsh-0.7_pre20160820::gentoo (masked by: package.mask)
- app-shells/scsh-0.6.7-r3::gentoo (masked by: package.mask)
```

Any guesses as to why?

Reply:
no idea

Reply:
So the ebuild was `0.7_pre20160820` and the mask was `0.7.0`. Take a look at https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-26008r2 and see if you can figure it out.

Reply:
the more "numeric components" the greater a package version would be considered,
so even if we mask `0.7.0`, `0.7` would be lower

Reply:
Right on point, awesome 👍

Reply:
Why this does this actually need multilib again?